cdc/sink/dispatcher/topic/dispatcher.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package topic
+
+import (
+	"github.com/pingcap/tiflow/cdc/model"
+)
+
+// Dispatcher is an abstraction for dispatching rows and ddls into different topics.
+type Dispatcher interface {
+	DispatchRowChangedEvent(row *model.RowChangedEvent) string
+	DispatchDDLEvent(ddl *model.DDLEvent) string
```

Comment:
Should we consider an `AllTopics` interface? Because checkpointts need to be broadcast to all topics, but this may not be a good place to implement it, as we can't determine the topics we need by expression alone, so if you find it difficult to add it now, we can add it later.

Reply:
I think we can add it later.